app/views/lessons/_lesson_buttons.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="flex flex-col justify-center max-w-sm mx-auto space-y-6 md:max-w-full md:space-y-0 md:flex-row md:space-x-7">
-  <%= link_to path_course_path(course.path, course), class: 'button button--secondary font-medium', data: { test_id: 'view-course-btn' } do %>
+  <%= link_to path_course_path(course.path, course), class: 'button button--clear font-medium', data: { test_id: 'view-course-btn' } do %>
```

Comment:
Nit: They look a tiny bit 🤏  too wide alongside the complete button. Reducing the padding sorts it out:

```suggestion
  <%= link_to path_course_path(course.path, course), class: 'button button--clear px-4 font-medium', data: { test_id: 'view-course-btn' } do %>
```

The next button will need this change too.

Reply:
Will do